Modules/_sqlite/connection.c

```diff
@@ -274,6 +274,11 @@ int pysqlite_connection_register_cursor(pysqlite_Connection* connection, PyObjec
 {
     PyObject* weakref;
 
+    if (!connection->cursors) {
+        PyErr_SetString(pysqlite_ProgrammingError, "Could not get the cursors list.");
```

Comment:
This can only happen if the Connection object is uninitialised; the error message should match the error condition. Also, PEP 7 disallows lines longer than 79 characters.

```suggestion
        PyErr_SetString(pysqlite_ProgrammingError,
                        "Base Connection.__init__ not called");
```